### PR TITLE
Document better usage with `execa` in readme

### PR DESCRIPTION
**Pull request by papb**
Hello!

The readme suggests calling `execa(binPath)`, but sadly this can give `EACCES` errors some times - happened to me right now in CI, although it worked on my machine (windows).

I believe this is because executing the `.js` file directly requires it to have `+x` permission in Linux. In windows, I would guess this is just another magic `execa` is doing under the hood and it works by luck.

In short, to avoid the risk of `+x` permission issues, I think using `execa.node` instead of `execa` is better.

It took me a while to figure this out :facepalm:

### CODE REVIEW

**Comment by papb** [comment]
:eyes:

**Comment by papb** [comment]
Hello, @ehmicky! Why did you close this?

**Comment by ehmicky** [comment]
Hi @papb,

I did not intend to! I accidentally deleted the branch, which closed to PR. Sorry! I am not sure how to bring it back... :(
I also did not get notified in June that this is PR was opened...
Sorry again!

Going back to the PR: I believe the real fix here would be to add `+x` permission to the binary file? If you have a package with a `bin` file without that file permission, you are most likely to encounter issues.

**Comment by papb** [comment]
No problem :)

Hmm I will test adding `+x` permission to it and let you know later.
